unblob/handlers/compression/lzo.py

```diff
@@ -75,7 +75,7 @@ class LZOHandler(StructHandler):
     """
     HEADER_STRUCT = "lzo_header"
 
-    EXTRACTOR = Command("lzop", "-d", "-f", "-N", "-p{outdir}", "{inpath}")
```

Comment:
just wondering how this could happen? the carved chunk will always have `.lzo` suffix. Or was it a whole file chunk without the proper suffix?

Reply:
Namely a file within an archive named 'ramdisk'.

Reply:
Should have read the original issue first ... :(